Source/WebCore/accessibility/AXTextMarker.cpp

```diff
@@ -1032,11 +1048,11 @@ AXTextMarkerRange AXTextMarker::sentenceRange(SentenceRangeType type) const
 
     if (type == SentenceRangeType::Current) {
         startMarker = previousSentenceStart();
-        endMarker = nextSentenceEnd();
+        endMarker = startMarker.nextSentenceEnd();
```

Comment:
Not in this change, but think we should have these functions, nextSentenceEnd() and similar, as standalone functions and not as AXTextMarker's methods.

Reply:
To me, these make sense as part of the AXTextMarker class, since they are types of movements you would want to make from an individual text marker. Are you suggesting that these become static on AXTextMarker, or completely independent?

Reply:
It's a matter of design, but I'd say these are operations you perform on a TextMarker, involving more knowledge that what a TextMarker has. So, I'd say they should be functions outside the class. But it is an arguable point.